flixel/addons/nape/FlxNapeSpace.hx

```diff
@@ -45,7 +45,7 @@ class FlxNapeSpace extends FlxBasic
 	public static var drawDebug(default, set):Bool;
 	
 	#if !FLX_NO_DEBUG
-	private static var shapeDebug:ShapeDebug;
+	public static var shapeDebug:ShapeDebug;
```

Comment:
Would be nice to have at least some basic documentation on what this is if it's made public.